api/views.py

```diff
@@ -27,38 +40,52 @@ def favorites(self, request, pk=None):
     favorites = user.grants.all()
     serializer = GrantSerializer(favorites, many=True)
     return Response(serializer.data)
-  
-  # @action(detail=True, methods=['post'])
-  # def favorites(self, request, pk=None, pk_grant=None):
-  #   user = self.get_object()
-  #   favorite = Grant.objects.get(pk=pk_grant)
-  #   user.grants.add(favorite)
-  #   serializer = GrantSerializer(favorite, many=False)
-  #   return JsonResponse(serializer.data)
+
 
 class GrantViewSet(viewsets.ModelViewSet):
   queryset = Grant.objects.all()
   serializer_class = GrantSerializer
 
   def list(self, request):
-    serializer = GrantSerializer(self.queryset, many=True)
+    education = request.GET.get('education','')
```

Comment:
Check out my comment on this PR for a possible refactor of checking query parameters this way! Not a reason to block merging this for now but for clean up purposes later :)